CHANGELOG.md

```diff
@@ -62,6 +62,8 @@ per cluster.
 - Add prompt for availability zone in pcluster configure automated subnets creation.
 - Add configuration HeadNode.Imds.Secured to enable/disable restricted access to IMDS.
 - Use different permissions in instance roles based on the scheduler and the node's role in the cluster.
+- Implement scaling protection mechanism with Slurm scheduler: compute fleet is automatically set to 'PROTECTED' state
+  in case recurrent failures are encountered when provisioning nodes.
```

Comment:
Should we mention resume timeout change here from cookbook? It seems to me a relevant change, but not 100% sure.